packages/office-ui-fabric-react/src/utilities/positioning/positioning.ts

```diff
@@ -889,3 +889,73 @@ export function getMaxHeight(
 export function getOppositeEdge(edge: RectangleEdge): RectangleEdge {
   return edge * -1;
 }
+
+/**
+ * Window with typings for experimental features regarding Dual Screen devices.
+ */
+export interface IWindowWithSegments extends Window {
```

Comment:
Should this be in types? And maybe we should keep this as internal for now until segments get added in more places?